Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml

```
@@ -0,0 +1,896 @@
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: efbb43c2-6320-4c3d-8c3f-6e716e4b8c8d
+    type: regular
+    task:
+      id: efbb43c2-6320-4c3d-8c3f-6e716e4b8c8d
+      version: -1
+      name: Run Prevalence Check On the Actor Process
+      description: Get the prevalence of a process, identified by process_name.
+      script: '|||core-get-process-analytics-prevalence'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "27"
+    scriptarguments:
+      process_name:
+        complex:
+          root: alert
+          accessor: osparentname
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 750,
+          "y": 240
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: aa11e09b-0a87-4a3e-8ffc-a40b503527d9
+    type: regular
+    task:
+      id: aa11e09b-0a87-4a3e-8ffc-a40b503527d9
+      version: -1
+      name: Search for suspicious-related alerts by MITRE Technique
+      description: |-
+        This task searches for Cortex XSIAM suspicious alerts related to the current incident by Mitre Technique, indicating that the alert is part of an attack pattern.
+
+        Focus on identifying alerts associated with the following MITRE techniques:
+        - Any Agent Alerts within this incident.
+        - T1059 - Command and Scripting Interpreter.
+      scriptName: SearchIncidentsV2
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "15"
+    scriptarguments:
+      query:
+        complex:
+          root: alert
+          accessor: parentXDRIncident
+          transformers:
+          - operator: Cut
+            args:
+              delimiter:
+                value:
+                  simple: '-'
+              fields:
+                value:
+                  simple: "2"
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: '((mitreattcktechnique:*T1509* or (sourceBrand:TRAPS and categoryname:Malware)) and caseid:'
+              suffix:
+                value:
+                  simple: )
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 960,
+          "y": 1290
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: b9bbfbad-b31e-4141-820d-447ae8cbb61a
+    type: condition
+    task:
+      id: b9bbfbad-b31e-4141-820d-447ae8cbb61a
+      version: -1
+      name: Any risky asset involved with a non-prevalent action?
```

```suggestion
      name: Any risky asset involved ?
```

at this step you are in a "prevalent" case and not in a "non-prevalent" case
anyway - no need to mention it the task. 

Please don't forget to update the pic after

---

Updated the name to `Any risky asset involved with at least one non-prevalent action?`